### PR TITLE
Let the repomap variable names explain themselves

### PR DESCRIPTION
**Pull request by bocekm**
It was not clear what `to_id` or `to_name` means and what's the difference - both are in fact repo names - one is the CDN repoid and the other is repo name as used in the PES.

### CODE REVIEW

**Comment by pirat89** [comment]
I see the point. But I don't like this naming either. Let's discuss it later whether we will come up with something.

File: repos/system_upgrade/el7toel8/models/repositoriesmap.py [hunk 1]
```diff
@@ -12,11 +12,11 @@ class RepositoryMap(Model):
 
     topic = TransactionTopic
 
-    from_id = fields.String()
-    to_id = fields.String()
-    to_name = fields.String()
-    from_minor_version = fields.String()
-    to_minor_version = fields.String()
+    from_repo_cdn = fields.String()  # RHEL 7 repoid as present in the Red Hat CDN
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/models/repositoriesmap.py]:
```suggestion
    from_repo_cdn = fields.String()
    """RHEL 7 repoid as present in the Red Hat CDN"""
    
```

**Comment by pirat89** [comment]
The https://github.com/oamg/leapp-repository/pull/359 PR has been merged. You need to apply changes in `repositoriesblacklisted` as well.

File: repos/system_upgrade/el7toel8/models/repositoriesmap.py (hunk 1 above)
Comment (bocekm) [reply repos/system_upgrade/el7toel8/models/repositoriesmap.py]: Done.

**Comment by bocekm** [comment]
`repositoriesblacklisted` actor changed too.
Changed:
`from_id` -> `from_repoid`
`to_id` -> `to_repoid`
`to_name` -> `to_pes_repo`

**Pull request merged**